src/provider/provider_os_memory_linux.c

```diff
@@ -28,6 +31,21 @@ umf_result_t os_translate_mem_visibility_flag(umf_memory_visibility_t in_flag,
     return UMF_RESULT_ERROR_INVALID_ARGUMENT;
 }
 
+// open or create a shared memory file
+int os_shm_open(const char *shm_name) {
+    if (shm_name == NULL) {
+        LOG_ERR("empty name of a shared memory file");
+        return -1;
+    }
+
+    int fd = shm_open(shm_name, O_RDWR, 0600);
```

Comment:
you wrote that the shared memory file must start with a ‘/’ character - could you check this here?

Reply:
It is checked inside `shm_open()` - see `shm_open(3)`:
` EINVAL The name argument to shm_open() was invalid.`